### ISSUE TITLE
bug in fetch debian bts

### ISSUE THREAD

**Issue by rickysarraf** [rickysarraf/apt-offline#44]
Downloading libc0.1 2.21-6 - 2 MiB                               B
Fetched bug reports for package libxml-libxml-perl                               
libxml-libxml-perl 2.0123+dfsg-1+b1 done                                                             
Downloading libc-bin 2.21-6 - 708 KiB                               
Ignoring err: Possibly multiarch package locales{}555168{}__apt__bug__report

Fetched bug reports for package libc0.1-dev                               
libc0.1-dev 2.21-6 done                                                             
Downloading libicu55 55.1-7 - 7 MiB                               
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineLib.py", line 636, in run
    self.responseQueue.put( self.WorkerFunction( item, thread_name ) )
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineCoreLib.py", line 965, in DataFetcher
    if FetchBugReportsDebian.FetchBugsDebian(PackageName) in [1,2]:
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineCoreLib.py", line 206, in FetchBugsDebian
    self.AddToArchive( self.ArchiveFile, self.fileName )
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineCoreLib.py", line 219, in AddToArchive
    os.unlink(fileName)
OSError: [Errno 2] No such file or directory: 'locales{}555168{}__apt__bug__report'

Fetched bug reports for package libc0.1                               
libc0.1 2.21-6 done                                                             
Downloading libxml2 2.9.3+dfsg1-1 - 867 KiB                               
Fetched bug reports for package libicu55                               
libicu55 55.1-7 done                                                             
Downloading liblz4-1 0.0~r131-1 - 31 KiB

**Comment by rickysarraf** [comment]
If it was handled as a multiarch package, why is that file still expected ?

**Comment by rickysarraf** [comment]
One more from the same instance.


```
Fetched bug reports for package apt-utils                               
apt-utils 1.2 done                                                             
Downloading libnettle6 3.1.1-4 - 184 KiB                               
Fetched bug reports for package libnettle6                               
libnettle6 3.1.1-4 done                                                             
Downloading libpsl0 0.11.0-2 - 39 KiB                               
Fetched bug reports for package libpsl0                               
libpsl0 0.11.0-2 done                                                             
Downloading wget 1.17.1-1 - 755 KiB                               
Ignoring err: Possibly multiarch package apt{}701564{}__apt__bug__report

Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineLib.py", line 636, in run
    self.responseQueue.put( self.WorkerFunction( item, thread_name ) )
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineCoreLib.py", line 1097, in DataFetcher
    if FetchBugReportsDebian.FetchBugsDebian( PackageName ) in [1, 2]:
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineCoreLib.py", line 206, in FetchBugsDebian
    self.AddToArchive( self.ArchiveFile, self.fileName )
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineCoreLib.py", line 219, in AddToArchive
    os.unlink(fileName)
OSError: [Errno 2] No such file or directory: 'apt{}701564{}__apt__bug__report'

Fetched bug reports for package perl                               
perl 5.22.1-4 done                                                             
Downloading libgnutls-deb0-28 3.3.20-1 - 704 KiB                               

```

**Comment by rickysarraf** [comment]
This needs to be fixed because I'm continuously seeing this error:

```
libperl5.22 copied from local cache directory /var/cache/apt/archives/                         Ignoring err: Possibly multiarch package libc6{}310445{}__apt__bug__report

Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineLib.py", line 636, in run
    self.responseQueue.put( self.WorkerFunction( item, thread_name ) )
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineCoreLib.py", line 1072, in DataFetcher
    if FetchBugReportsDebian.FetchBugsDebian(PackageName) in [1,2]:
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineCoreLib.py", line 206, in FetchBugsDebian
    self.AddToArchive( self.ArchiveFile, self.fileName )
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineCoreLib.py", line 219, in AddToArchive
    os.unlink(fileName)
OSError: [Errno 2] No such file or directory: 'libc6{}310445{}__apt__bug__report'

Fetched bug reports for package libc-bin                                                             
libc-bin copied from local cache directory /var/cache/apt/archives/                            Fetched bug reports for package libc6-dev                                                             
libc6-dev copied from local cache directory /var/cache/apt/archives/                           Fetched bug reports for package perl-base                                                             
perl-base copied from local cache directory /var/cache/apt/archives/                           Fetched bug reports for package libterm-readkey-perl                                                             
libterm-readkey-perl copied from local cache directory /var/cache/apt/archives/                Fetched bug reports for package liblocale-gettext-perl                                                             
liblocale-gettext-perl copied from local cache directory /var/cache/apt/archives/              Fetched bug reports for package libcurses-perl                                                             
libcurses-perl copied from local cache directory /var/cache/apt/archives/                      Fetched bug reports for package libtext-charwidth-perl                                                             
libtext-charwidth-perl copied from local cache directory /var/cache/apt/archives/              Fetched bug reports for package libtinfo5                                                             
libtinfo5 copied from local cache directory /var/cache/apt/archives/                           Fetched bug reports for package libtext-iconv-perl                                                             
libtext-iconv-perl copied from local cache directory /var/cache/apt/archives/                  Fetched bug reports for package libncurses5                                                             
libncurses5 copied from local cache directory /var/cache/apt/archives/                         Fetched bug reports for package libncursesw5                                                             
libncursesw5 copied from local cache directory /var/cache/apt/archives/                        Fetched bug reports for package debconf-i18n                                                             
debconf-i18n copied from local cache directory /var/cache/apt/archives/                        Fetched bug reports for package locales                                                             
locales copied from local cache directory /var/cache/apt/archives/                             Ignoring err: Possibly multiarch package linux-libc-dev{}522773{}__apt__bug__report

Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineLib.py", line 636, in run
    self.responseQueue.put( self.WorkerFunction( item, thread_name ) )
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineCoreLib.py", line 1072, in DataFetcher
    if FetchBugReportsDebian.FetchBugsDebian(PackageName) in [1,2]:
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineCoreLib.py", line 206, in FetchBugsDebian
    self.AddToArchive( self.ArchiveFile, self.fileName )
  File "/media/SSHD/rrs-home/devel/apt-offline/apt-offline/apt_offline_core/AptOfflineCoreLib.py", line 219, in AddToArchive
    os.unlink(fileName)
OSError: [Errno 2] No such file or directory: 'linux-libc-dev{}522773{}__apt__bug__report'


```

**Comment by rickysarraf** [comment]
This doesn't seem to be duplicate report for the same package. Looks like FetchBugReportsDebian is giving wrong status

**Comment by rickysarraf** [comment]
Should be fixed with: 4deac068e2dfedb0d9628af9b2be0d00e5608469
